example/bazel.repos

```diff
@@ -22,7 +22,7 @@ repositories:
   msgs:
     type: git
     url: https://github.com/gazebosim/gz-msgs
-    version: gz-msgs9
+    version: mjcarroll/garden_bazel
```

Comment:
create a PR for this branch?

Reply:
https://github.com/gazebosim/gz-msgs/pull/406